packages/app/cypress/integration/components/search/SearchPage.js

```diff
@@ -65,40 +95,24 @@ describe('SearchPage', () => {
           },
         },
       );
-      cy.contains('Search');
+      cy.expect(Heading('Search').exists());
 
       // lifecycle
-      cy.contains('lifecycle');
-
-      cy.contains('experimental');
-      cy.get(
-        '[data-testid="search-checkboxfilter-next"] input[value="experimental"]',
-      ).should('have.attr', 'checked');
-
-      cy.contains('production');
-      cy.get(
-        '[data-testid="search-checkboxfilter-next"] input[value="production"]',
-      ).should('not.have.attr', 'checked');
+      cy.expect(FormControl('Lifecycle').exists());
+      cy.expect(Checkbox('experimental').exists());
+      cy.expect(Checkbox('production').exists());
 
       // kind
-      cy.contains('kind');
-      cy.get(
-        '[data-testid="search-selectfilter-next"] [role="button"][aria-haspopup="listbox"]',
-      ).click();
-
-      cy.contains('All');
-      cy.contains('Template');
-      cy.contains('Component');
+      cy.do(Select('Kind').open());
 
-      cy.get('[role="option"][data-value="Component"]').should(
-        'have.attr',
-        'aria-selected',
-        'true',
+      cy.expect(Popover().find(SelectOption('All')).exists);
+      cy.expect(Popover().find(SelectOption('Template')).exists());
+      cy.expect(
+        Popover().find(SelectOption('Component')).is({ selected: true }),
```

Comment:
Do we need `Popover.find()` here? Is that not specific enough? In other words would `SelectOption('All')` alone work?

Reply:
Done, it works